.github/workflows/linux_ubuntu.yml

```diff
@@ -32,23 +34,18 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        cubit: [2021.4, 2021.5, 2021.11]
+        cubit: [2021.5, 2021.11]
         os: [ubuntu]
         os_version: [20.04, 21.04]
         include:
           - os: ubuntu
             os_version: 18.04
             cubit: 17.1.0
 
-          - os: debian
-            os_version: '10.10'  # using a 'string' here as the 0 gets rounded away otherwise
-            cubit: 2021.5
-
-          - os: debian
-            os_version: '10.10'  # using a 'string' here as the 0 gets rounded away otherwise
-            cubit: 2021.11
+          - os: ubuntu
```

Comment:
this has now excluded the U21.04 + C2021.4 combination - was that failing before?

Reply:
I think it was not failing before. I can't quite figure out why it is failing now.

> Thanks for the work here @shimwell - I wonder how to deal with third party actions becoming unsupported in the long term?

I think the newer replacement are usually easier to use which I guess is part of the reason they become the preferred option. They also tend to offer the same or better functionality. So perhaps upgrading actions won't be much trouble. This action repo was kind enough to recommend  a replacement one.

Reply:
> this has now excluded the U21.04 + C2021.4 combination - was that failing before?

I don't quite understand why it no longer works but it seems that U21.04 + C2021.4 was working previously. I can manually upload it to the release of that helps